### PR TITLE
Apply yaml-lint to Github workflows and actions

### PR DESCRIPTION
**Pull request by kerbou**
Ref: https://app.zenhub.com/workspaces/the-outlaws-6193fe815d79fc0011e741b1/issues/gh/energinet-datahub/the-outlaws/611

### CODE REVIEW

File: .github/actions/github-actions-input-name-validate/action.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  using: composite
+  steps:
+
+    - name: Github input variables should be lowercase
+      if: always()
+      shell: pwsh
+      run: |
+        $folder = "${{ inputs.yaml_file_or_folder }}"
+
+        Install-Module -Name powershell-yaml -Force -Repository PSGallery -Scope CurrentUser
```
Comment (djorgensendk) [inline .github/actions/github-actions-input-name-validate/action.yaml:37]: can you try without: -Repository PSGallery -Scope CurrentUser

File: .github/actions/github-actions-input-name-validate/action.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        Write-Host "Files found in $($folder): $($files.Length)"
+        $uppercaseVarsFound = $false
+
+        foreach($file in $files) {
+            Write-Host "Checking $($file.FullName)"
+            $yaml = Get-Content -Path $file.FullName | Out-String
+            $yaml = $yaml.Replace('{{', '').Replace('}}', '')
+            $jsonObj = (ConvertFrom-Yaml -Yaml $yaml)
+
+            foreach($inputKey in ($null -ne $jsonObj.on.workflow_call.inputs ? $jsonObj.on.workflow_call.inputs.Keys : $jsonObj.inputs.Keys)) {
```
Comment (djorgensendk) [inline .github/actions/github-actions-input-name-validate/action.yaml:48]: can : $jsonObj.on.workflow_call.inputs ? $jsonObj.on.workflow_call.inputs.Keys : $jsonObj.inputs.Keys be assigned to a meaningful variabel?

the $null -ne check - can that be done in an enclosing if-statement instead?

File: .github/actions/github-actions-input-name-validate/action.yaml (hunk 1 above)
Comment (kerbou) [reply .github/actions/github-actions-input-name-validate/action.yaml]: Done - works fine

File: .github/actions/github-actions-input-name-validate/action.yaml (hunk 2 above)
Comment (kerbou) [reply .github/actions/github-actions-input-name-validate/action.yaml]: 1. Yes, done
2. I've learned that Powershell 7 has implemented null-coalescing, so using that removes unnecessary boilerplate code

**Review by djorgensendk — APPROVED** [review]

**Pull request merged**